src/handlers/gestures/gesture.ts

```diff
@@ -24,7 +24,9 @@ export type GestureType =
   | BaseGesture<ForceTouchGestureHandlerEventPayload>;
 
 export type GestureRef = number | GestureType | React.RefObject<GestureType>;
-export interface BaseGestureConfig extends CommonGestureConfig {
+export interface BaseGestureConfig
+  extends CommonGestureConfig,
+    Record<string, unknown> {
```

Comment:
Why do we extend Record here?

Reply:
Because `filterConfig` expects config to be a `Record`.